pkg/auth/api/user/user_actions.go

```diff
@@ -170,3 +180,22 @@ func (h *Handler) refreshAttributes(actionName string, action *types.Action, req
 func (h *Handler) userCanRefresh(request *types.APIContext) bool {
 	return request.AccessControl.CanDo(v3.UserGroupVersionKind.Group, v3.UserResource.Name, "create", request, nil, request.Schema) == nil
 }
+
+func getMinimumPasswordLength() int {
+	return settings.PasswordMinLength.GetInt()
+}
+
+// validatePassword will ensure a password is the minimum required length, and that the password does not contain the username.
```

Comment:
The comment "the password does not contain the username" doesn't match what the func is doing - it's checking if they are not exactly the same.